pages/linux/i3exit.md

```diff
@@ -0,0 +1,32 @@
+# i3exit
+
+> Exit the i3 window manager.
+> More information: <https://gitlab.manjaro.org/packages/community/i3/i3exit>.
+
+- Log out of i3:
+
+`i3exit logout`
+
+- Lock i3:
+
+`i3exit lock`
+
+- Shut down the machine:
+
+`i3exit shutdown`
+
+- Suspend the machine:
+
+`i3exit suspend`
+
+- Switch the current user:
```

Comment:
Switch to what? :thinking:

Reply:
I logs you out and prompts you to choose a user. It's the same as in every DE when you click on "switch user".

Reply:
It's not the same goes to the select user screens keeping the current session open, than log off and go to the select user to login screen. It's nice to leave that clear here.

Reply:
Well, I'm not actually sure. I tried it, but it just showed me a blank screen and crashed. But what it does is calling `dm-tool switch-to-greeter`. From the man pages

> Switch to the greeter suitable for logging into a new session.

Sounds like it doesn't log you out. How about we change the description to
```suggestion
- Switch to the login screen to log in as a different user:
```

Reply:
It's sounds like this is essentially a "switch user" feature. @marchersimon's suggestion seems logical to me.